v1.1.0 "make contracts" fails

```
# git checkout dusk-rusk-1.1.0
# make contracts
make -j1 -C ./contracts all
make[1]: Entering directory '/root/rusk-build/contracts'
make -C alice all
make[2]: Entering directory '/root/rusk-build/contracts/alice'
error: "/root/rusk/target/dusk/compiler/v0.2.0/extracted/lib/rustlib/src/rust/Cargo.lock" does not exist, unable to build with the standard library, try:
        rustup component add rust-src --toolchain dusk
make[2]: *** [Makefile:6: wasm] Error 101
make[2]: Leaving directory '/root/rusk-build/contracts/alice'
make[1]: *** [Makefile:19: alice] Error 2
make[1]: Leaving directory '/root/rusk-build/contracts'
make: *** [Makefile:22: contracts] Error 2
```

```
# rustc -V
rustc 1.75.0-nightly (0f44eb32f 2023-11-09)
```

Does not compile

--- a/Makefile
+++ b/Makefile
@@ -18,7 +18,7 @@ wasm: setup-compiler ## Generate the WASM for all the contracts and wallet-core
 	$(MAKE) -C ./contracts $@
 	$(MAKE) -C ./wallet-core $@
 
-contracts: ## Execute the test for all contracts
+contracts: setup-compiler ## Execute the test for all contracts
 	$(MAKE) -j1 -C ./contracts all
 
 test: keys wasm ## Run the tests

